### ISSUE TITLE
Feature Request: Add  “stone” style 

### ISSUE THREAD

**Issue by Ali-RS** [jMonkeyEngine-Contributions/Lemur#78]
What about to include  “stone” style for the next release :)

**Comment by pspeed42** [comment]
Yeah, could be time.  I have to look through to see if it's complete.

I wonder if it would be better to package extra styles as separate jars?

**Comment by Ali-RS** [comment]
> I wonder if it would be better to package extra styles as separate jars?
Seems good idea. 
And then later on, for each style maybe we can also provide a pure java version along with groovy version, to make Lemur more android friendly.
